docs/people-and-groups/changing-password-complexity.md

```diff
@@ -24,6 +24,12 @@ The settings above can be used independently, so it's fine to use only one or th
 By default, Metabase also prevents users from setting passwords that are in a list of common passwords (like `qwerty123` and
 `passw0rd`). Changing the complexity requirement to `weak` disables this behavior.
 
+## Metabase Cloud passwords
+
+If your instance is hosted on Metabase Cloud, the follwing password compexity rules apply:
+- passwords need to be at least 12 characters long
+- passwords must contain at least 2 lowercase, 2 uppercase, 1 digit, and 1 special character
```

Comment:
```suggestion
If your instance is hosted on Metabase Cloud, passwords must:

- Be at least 12 characters long
- Contain at least: 
  - 2 lowercase
  - 2 uppercase
  - 1 digit
  - 1 special character
```

Edit strips out redundant text "Passwords need to be, passwords must contain". And renders a list so reqs are easier to scan. 

(Might need to run prettier on the formatting)